packages/grafana-data/src/transformations/transformers/convertFieldType.ts

```diff
@@ -142,7 +142,9 @@ function fieldToNumberField(field: Field): Field {
   const numValues = field.values.toArray().slice();
 
   for (let n = 0; n < numValues.length; n++) {
-    const number = +numValues[n];
+    // some numbers returned from datasources have commas
+    // strip the commas, coerce the string to a number
+    const number = +numValues[n].replace(/,/g, '');
```

Comment:
thought that skipping the regex might be faster, but it wasn't :)

https://jsfiddle.net/sga5nwpt/1/

i assume we dont have to handle locales where `,` is a decimal separator and `.` is the thou separator?

Reply:
This is a good point! We could always add a toggle for convert field to number, and let the user select Imperial or Metric.